src/runtime/CMakeLists.txt

```diff
@@ -339,3 +339,6 @@ add_library(Halide::Runtime ALIAS Halide_Runtime)
 target_include_directories(Halide_Runtime INTERFACE $<BUILD_INTERFACE:${Halide_BINARY_DIR}/include>)
 set_target_properties(Halide_Runtime PROPERTIES EXPORT_NAME Runtime)
 
+if (TARGET_HEXAGON)
```

Comment:
Since this condition can be true on builders that do not handle hexagon, maybe add another check to see if HEXAGON_SDK_ROOT is set up in the environment. It is not the most elegant thing to do though.

Reply:
@alexreinking @steven-johnson - Any thoughts on this?

Reply:
A simpler solution was to simply not build the hexagon backend in llvm on builders that don't handle hexagon. I have updated my PR -> https://github.com/halide/build_bot/pull/237